### PR TITLE
Position details - new design

### PR DESCRIPTION
**Pull request by NataliaJurecka**

### CODE REVIEW

File: src/components/PositionDetails/LiquidationRangeInfo/style.ts [hunk 1]
```diff
@@ -1,22 +1,23 @@
 import { makeStyles, Theme } from '@material-ui/core/styles'
-import { colors, typography } from '@static/theme'
+import { colors, newTypography, typography } from '@static/theme'
 
 const useStyles = makeStyles((theme: Theme) => ({
   infoTypeLabel: {
     textTransform: 'uppercase',
-    color: colors.invariant.lightInfoText,
-    ...typography.body1,
+    color: '#A9B6BF',
```
Comment (awojciak) [inline src/components/PositionDetails/LiquidationRangeInfo/style.ts:7]: use colors from theme

Comment (awojciak) [reply src/components/PositionDetails/LiquidationRangeInfo/style.ts:7]: comment about all uses of colors

File: src/components/PositionDetails/SinglePositionInfo/style.ts [hunk 2]
```diff
@@ -137,54 +139,82 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+    backgroundColor: '#111931',
+    borderRadius: '16px',
     flexDirection: 'row',
     width: '100%',
     justifyContent: 'space-between',
+    padding: 12,
     '&:not(:last-child)': {
       marginBottom: 8
     }
+    // '&:not(:first-child)': {
```
Comment (awojciak) [inline src/components/PositionDetails/SinglePositionInfo/style.ts:157]: remove this commented code

File: src/components/PriceRangePlot/Brush/Brush.tsx [hunk 3]
```diff
@@ -83,15 +83,15 @@ export const Handle: React.FC<HandleProps> = ({
         <MinHandle
           height={height}
           x={!isReversed() ? currentPosition - 37 : currentPosition}
-          fill={disabled ? colors.invariant.componentOut3 : colors.invariant.accent1}
+          fill={disabled ? colors.invariant.light : colors.invariant.light}
```
Comment (awojciak) [inline src/components/PriceRangePlot/Brush/Brush.tsx:86]: these colors shouldn't be the same on enabled and disabled state, check colors on enabled on mockups for adding positions

File: src/components/PriceRangePlot/PriceRangePlot.tsx [hunk 4]
```diff
@@ -278,7 +283,7 @@ export const PriceRangePlot: React.FC<IPriceRangePlot> = ({
         ]}
         curve='basis'
         margin={{ top: isSmDown ? 55 : 25, bottom: 15 }}
-        colors={[colors.invariant.accent1, colors.invariant.accent2, colors.invariant.accent1]}
+        colors={[colors.invariant.green, colors.invariant.green, colors.invariant.green]}
```
Comment (awojciak) [inline src/components/PriceRangePlot/PriceRangePlot.tsx:286]: color for elements 0 and 2 should be pink

**Review by awojciak — CHANGES_REQUESTED** [review]
also use smaller margins between icons on xs breakpoint
obraz

**Review by Pawel-Szydlo — COMMENTED** [review]
obraz

  text hides under button
- Maybe you should change the "Close position" button font, check it on old mockups.

File: src/components/PositionDetails/LiquidationRangeInfo/style.ts (hunk 1 above)
Comment (NataliaJurecka) [reply src/components/PositionDetails/LiquidationRangeInfo/style.ts]: change done, new definition od color "textGrey" changed in the theme

File: src/components/PositionDetails/SinglePositionInfo/style.ts (hunk 2 above)
Comment (NataliaJurecka) [reply src/components/PositionDetails/SinglePositionInfo/style.ts]: commented code deleted

File: src/components/PriceRangePlot/PriceRangePlot.tsx (hunk 4 above)
Comment (NataliaJurecka) [reply src/components/PriceRangePlot/PriceRangePlot.tsx]: done

File: src/components/PriceRangePlot/Brush/Brush.tsx (hunk 3 above)
Comment (NataliaJurecka) [reply src/components/PriceRangePlot/Brush/Brush.tsx:85]: done

**Review by mlodyFrontendowiec — CHANGES_REQUESTED** [review]
image
In my opinion it is too big, it should be row direction between 960px and 600px window width

**Pull request merged**